src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -0,0 +1,269 @@
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(-12345);
+            standardOperators.BackSpaceButton.Click();
+            Assert.AreEqual("-1,234", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void BVT_ClearAll()
+        {
+            standardOperators.Add(new List<double>() { 12345, 6789 });
+            standardOperators.ClearButton.Click();
+            Assert.AreEqual("0", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void BVT_ClearEntry()
+        {
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(-12345);
+            standardOperators.MinusButton.Click();
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(5678);
+            standardOperators.ClearEntryButton.Click();
+            Assert.AreEqual("0", StandardCalculatorPage.GetCalculatorResultText());
+        }
+        #endregion
+
+        #region Keyboard Input Tests
+        [TestMethod]
+        public void KeyboardInput_Subtract()
+        {
+            StandardCalculatorPage.SendKeys("70090-890987=");
+            Assert.AreEqual("-820,897", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_Multiply()
+        {
+            StandardCalculatorPage.SendKeys("-15*3=");
+            Assert.AreEqual("-45", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_Divide()
+        {
+            StandardCalculatorPage.SendKeys("26/13=");
+            Assert.AreEqual("2", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_Reciprocal()
+        {
+            StandardCalculatorPage.SendKeys("10r");
+            Assert.AreEqual("0.1", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_Squared()
+        {
+            StandardCalculatorPage.SendKeys("3q");
+            Assert.AreEqual("9", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_SquareRoot()
+        {
+            StandardCalculatorPage.SendKeys("100@");
+            Assert.AreEqual("10", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_Cubed()
+        {
+            StandardCalculatorPage.SendKeys("3#");
+            Assert.AreEqual("27", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void KeyboardInput_Percent()
+        {
+            StandardCalculatorPage.SendKeys("34+7%=");
+            Assert.AreEqual("36.38", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [Ignore]
+        [TestMethod]
+        public void KeyboardInput_Delete()
+        {
```

Comment:
Don't check in dead code unless we have a bug tracking getting these working again (or Assert.Inconclusive(), or something else which is harder to forget about)